starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -986,16 +986,15 @@ private int queueSecureInputBuffer(
                 + "CryptoException.ERROR_INSUFFICIENT_OUTPUT_PROTECTION");
         return MediaCodecStatus.INSUFFICIENT_OUTPUT_PROTECTION;
       }
-      Log.e(
-          TAG,
-          "Failed to queue secure input buffer, CryptoException with error code "
-              + e.getErrorCode());
+      Log.e(TAG, "Failed to queue secure input buffer. Error code %d", errorCode, e);
       return MediaCodecStatus.ERROR;
     } catch (IllegalArgumentException e) {
```

Comment:
MediaCodec.CodecException is caught on the upstream, consider catching it here too if it can be thrown by the function.